api/envoy/extensions/filters/http/router/v3/router.proto

```diff
@@ -78,4 +78,9 @@ message Router {
   // :ref:`config_http_filters_router_x-envoy-expected-rq-timeout-ms` header, populated by egress
   // Envoy, when deriving timeout for upstream cluster.
   bool respect_expected_rq_timeout = 6;
+
+  // If set, Envoy will avoid incrementing HTTP failure code stats
+  // on gRPC requests. This includes the individual status code value
+  // (e.g. upstream_rq_504) and group stats (e.g. upstream_rq_5xx).
```

Comment:
Can you add a link to the gRPC stats filter and comment on why one might want to set this? Also, conversely, can you add a link from the gRPC stats filter to this field and describe why someone might want to set it?